internal/pkg/manifest/env_test.go

```diff
@@ -873,3 +924,63 @@ func TestEnvironmentCDNConfig_CDNEnabled(t *testing.T) {
 		})
 	}
 }
+
+func TestEnvironmentAccessLogs_ELBAccessLogs(t *testing.T) {
```

Comment:
nit: so that it starts with the receiver type's name, and then the method name
```suggestion
func TestEnvironmentConfig_ELBAccessLogs(t *testing.T) {
```

Reply:
Ohh got it.